pkg/metrics/constants.go

```diff
@@ -27,6 +27,7 @@ const (
 	ProvisionerLabel = "provisioner"
 	NodePoolLabel    = "nodepool"
 	ReasonLabel      = "reason"
+	TypeLabel        = "type"
```

Comment:
This is specific to disruption so does this need to be in the generic metrics package?

Reply:
If we move this into the disruption controller, we would have a dependency cycle